src/components/vr-scatter-plot/vr-scatter-plot.component.ts

```diff
@@ -76,7 +83,7 @@ export class VRScatterPlotComponent<T extends Meta>implements OnInit, OnChanges,
   }
 
   isTabbed(): boolean {
-    return 'tabbed' === this.meta.type;
+    return 'tabbed' === (this.meta as Meta).type;
```

Comment:
What are other possible values for the type? Could 'tabbled' be a string or enum?

Reply:
gonna go ahead and get rid of this function. Instead I will check the type in the function within an `if / else if` structure -- within this branch there is only type `line` and type `tabbed`. I am making a new PR that has added a dataset and added a type `vrScatter`

In response to your question, do you mean checking every case of an enum that includes all the types?